components/brave_ads/browser/ads_status_header_throttle_unittest.cc

```diff
@@ -17,16 +18,21 @@ namespace {
 
 constexpr char kAdsStatusHeader[] = "X-Brave-Ads-Enabled";
 constexpr char kAdsEnabledStatusValue[] = "1";
-constexpr char kAllowedURL[] = "https://search.brave.com/search";
-constexpr char kNotAllowedURL[] = "https://brave.com/search";
+constexpr const char* kAllowedHosts[] = {
```

Comment:
Would it make sense to re-use `kVettedHosts` here?

Reply:
`kVettedHosts` is a fixed_flat_set and declaring it in a header will be ugly:
```
template <class Key, size_t N, class Compare = std::less<>>
using fixed_flat_set = base::flat_set<Key, Compare, std::array<const Key, N>>;

```

Reply:
the issue would be that if we duplicate these lists then a future change could make `kVettedHosts` and `kAllowedHosts` become out of sync. It is ugly but how often would we need to make changes to the above template?

Reply:
It won't change often. It is the first change in 1.5 years.